sale_stock_picking_note/views/report_deliveryslip.xml

```diff
@@ -0,0 +1,14 @@
+<odoo>
+    <template
+        id="report_delivery_document_comments"
+        inherit_id="stock.report_delivery_document"
+    >
+        <xpath expr="//table[@name='stock_move_line_table']" position="after">
+            <p t-if="o.comments">
```

Comment:
This hasn't been changed. You can export to HTML the report in the tests as a little check.